### PR TITLE
align metrics

### PR DESCRIPTION
**Pull request by kprokofi**
[CVS-94144](https://jira.devtools.intel.com/browse/CVS-94144)

### CODE REVIEW

**Comment by kprokofi** [comment]
run ote-test

File: external/model-preparation-algorithm/mpa_tasks/extensions/datasets/mpa_cls_dataset.py [hunk 1]
```diff
@@ -128,7 +128,12 @@ def evaluate(self,
             metrics.remove('class_accuracy')
             self.class_acc = True
 
+        # compute top-k metrics from mmcls and align them in [0,1] range
         eval_results = super().evaluate(results, metrics, metric_options, logger)
+        for k in metric_options['topk']:
+            if 0 <= eval_results[f'accuracy_top-{k}'] <= 100:
```
Comment (sovrasov) [inline external/model-preparation-algorithm/mpa_tasks/extensions/datasets/mpa_cls_dataset.py]: top-k score is always <= 100. Isn't it?

Comment (kprokofi) [reply external/model-preparation-algorithm/mpa_tasks/extensions/datasets/mpa_cls_dataset.py]: I'm wrong. [0, 1] included in [0, 100]. We should check case [ 1, inf)

**Comment by kprokofi** [comment]
run ote-test

**Comment by kprokofi** [comment]
http://validationreports.sclab.intel.com:8006/reports/test_results/638155f77b4ed0f4e39e61e4

for classification only hpo tests failed, that I guess is expected

**Review by supersoob — DISMISSED** [review]
LGTM

**Review by sovrasov — DISMISSED** [review]

**Review by supersoob — COMMENTED** [review]
It looks good but I'm confused if its base branch is okay to be [releases/v0.3.1-geti1.0.0](https://github.com/openvinotoolkit/training_extensions/tree/releases/v0.3.1-geti1.0.0). @goodsong81 Didn't we wrap up the release branch? I thought we should fix the bugs in develop branch from now on even it's reported from geti1.0.0.

**Review by JihwanEom — DISMISSED** [review]
LGTM, but same opinion with Soobee. Let's wait Songki's comment.

**Review by goodsong81 — CHANGES_REQUESTED** [review]
Yeah, this PR needs to be retargeted to develop branch, and you may want to change the fix version in the ticket to Geti1.1/OTE0.4.0.

**Review by goodsong81 — APPROVED** [review]
LGTM. Thank you!

**Review by sovrasov — APPROVED** [review]

**Pull request merged**